llm/generate/gen_common.sh

```diff
@@ -61,6 +61,10 @@ apply_patches() {
     if ! grep ollama ${LLAMACPP_DIR}/examples/server/CMakeLists.txt; then
         echo 'include (../../../ext_server/CMakeLists.txt) # ollama' >>${LLAMACPP_DIR}/examples/server/CMakeLists.txt
     fi
+
+    # apply temporary patches until fix is upstream
+    (cd ${LLAMACPP_DIR} && git apply ../patches/server-kv-rm.diff)
```

Comment:
Not sure how long we'll need to carry this, but probably makes sense to add it to `gen_windows.ps1` as well.

This happens to work for repeated builds as long as things don't fail, since the patch only applies to server.cpp which we're already reverting in `cleanup` but if the patch touched other files, or the generate fails for any reason it'll get wedged.  I'm wondering if we should invest in a bit more robust rigging for this so we can apply patches in order, and then revert them in the reverse order, and ensure its idempotent.   For now this is probably OK, but if we find we're needing to do this ~often, I can look at some more plumbing to facilitate.